plugins/redfish/fu-redfish-backend.c

```diff
@@ -468,13 +526,20 @@ fu_redfish_backend_get_push_uri_path(FuRedfishBackend *self)
 	return self->push_uri_path;
 }
 
+const gchar *
+fu_redfish_backend_get_session_key(FuRedfishBackend *self)
+{
+	return self->session_key;
+}
+
 static void
 fu_redfish_backend_to_string(FuBackend *backend, guint idt, GString *str)
 {
 	FuRedfishBackend *self = FU_REDFISH_BACKEND(backend);
 	fwupd_codec_string_append(str, idt, "Hostname", self->hostname);
 	fwupd_codec_string_append(str, idt, "Username", self->username);
 	fwupd_codec_string_append_bool(str, idt, "Password", self->password != NULL);
+	fwupd_codec_string_append_bool(str, idt, "SessionKey", self->session_key != NULL);
```

Comment:
maybe just print it? this is for debugging really, so it's probably useful. e.g. use fwupd_codec_string_append instead.